cranelift/filetests/filetests/runtests/issue5714.clif

```diff
@@ -0,0 +1,23 @@
+test interpret
+test run
+target riscv64gc
```

Comment:
Would you be able to add targets for the other architectures? Even though this issue didn't impact them, It's still a useful regression test!

Reply:
sure.